pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -1445,6 +1448,13 @@ type ContainerRuntime struct {
 	ProviderConfig *runtime.RawExtension `json:"providerConfig,omitempty" protobuf:"bytes,2,opt,name=providerConfig"`
 }
 
+// WorkersSettings contains settings for all workers.
+type WorkersSettings struct {
+	// EnsureSSHAccessDisabled indicates whether the worker nodes ssh access is disabled.
```

Comment:
Maybe this description of the field sounds a little better. WDYT?

```suggestion
	// EnsureSSHAccessDisabled indicates whether the ssh access to the worker nodes should be disabled.
```